### PR TITLE
DX-2820 Cypress Tests for Tab Switching and API Reference Pages

### PR DESCRIPTION
**Pull request by brianluisgomez**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by ajrice6713 — APPROVED** [review]

File: cypress/e2e/tests/docs_sidebar.cy.js [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+      .should('have.attr', 'aria-expanded') 
+      .and('match', /true/)
+    cy.get('a.menu__link')
+      .contains("911 Access Dashboard")
+      .click()
+      .should('have.attr', 'aria-expanded') 
+      .and('match', /false/)    
+    })
+})
+
```
Comment (ckoegel) [inline cypress/e2e/tests/docs_sidebar.cy.js:470]: should have an empty newline at the end

File: cypress/e2e/tests/api_reference_download_button.cy.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  describe('Verify download button exists in the Insights API Reference Page', () => {
+    it('checks the download button to verify it exists',() => {
+      cy.visit('/apis/insights')
+      cy.get('p')
+        .contains("Download OpenAPI specification")
+        .children('a')
+        .contains("Download")      
+      })
+  })
+
```
Comment (ckoegel) [inline cypress/e2e/tests/api_reference_download_button.cy.js:92]: should have an empty newline at the end

File: cypress/e2e/tests/docs_sidebar.cy.js (hunk 1 above)
Comment (brianluisgomez) [reply cypress/e2e/tests/docs_sidebar.cy.js]: fixed

File: cypress/e2e/tests/api_reference_download_button.cy.js (hunk 2 above)
Comment (brianluisgomez) [reply cypress/e2e/tests/api_reference_download_button.cy.js]: fixed

**Review by ckoegel — APPROVED** [review]

**Pull request merged**